common/datablocks/src/kernels/data_block_except.rs

```diff
@@ -0,0 +1,41 @@
+// Copyright 2021 Datafuse Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use common_exception::ErrorCode;
+use common_exception::Result;
+
+use crate::DataBlock;
+
+// Compute the difference of two DataBlocks.
+impl DataBlock {
+    pub fn except_blocks(first_block: &DataBlock, second_block: &DataBlock) -> Result<DataBlock> {
```

Comment:
Can be improved by SortMerge.

Reply:
I'm replacing `Vec` with `HashSet`, then directly use https://doc.rust-lang.org/std/collections/struct.HashSet.html#method.difference

Reply:
Oh, It suddenly occurred to me that this function is no longer needed and I will delete it 🤣